ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -316,6 +320,9 @@ measurement:
   voltageratio: "The ratio of the Primary voltage of the transformer to the secondary voltage."
   volume: "The quantity of three-dimensional space contained by a closed surface."
   volumetricconcentration: "Concentration in terms of volume (parts per unit volume)."
+  xvalue: "X-value of a matrix"
+  yvalue: "Y-value of a matrix"
+  hours: "Measurement of elapsed time in hours"
```

Comment:
use `time` instead, with units `hours`

Reply:
Either way or the other, it throws an error.: If I don't tag them as "STANDARD", the validator in Git throws an error. If Io so, as documented in the picture, my ontology_validator throws one. Any ideas?

x-special/nautilus-clipboard
copy
file:///tmp/VMwareDnD/gl8cUz/Units_yaml.png

Reply:
![image](https://user-images.githubusercontent.com/82176022/205059013-89d79619-32c6-483c-b931-d31b3e9f9f5e.png)